### ISSUE TITLE
Can't open images

### ISSUE THREAD

**Issue by mattnoir** [sindresorhus/caprine#2240]
Seems like messenger changed something again and I am not able to open pictures in messages, nothing happens when I click on it.

Also, seems like Caprine is done, last modification 7 months ago, hundreds of issues open, several pull requests open as well. Is there some client that works in a similar way and is not abandoned? Thanks

**Comment by carneeki** [comment]
+1 - I first noticed images would not open two or three days ago, I thought it was just me.

**Comment by SilverSaw** [comment]
This is due to hide-preferences-window workaround listed here https://github.com/sindresorhus/caprine/issues/2167 , but since 2 days caprine has been forcing hide-preferences-window automatically on every restart. We are unable to manually change this permanently, and the bug is critical, on every restart we have to remove it manually to make caprine work.

**Comment by thankarezos** [comment]
> This is due to `hide-preferences-window` bug, a workaround is listed here #2167 , but since 2 days caprine has been forcing `hide-preferences-window` automatically on every restart. We are unable to manually change this permanently, and the bug is critical, on every restart we have to remove it manually to make caprine work.

I found a workaround for that too. Go to file>caprine settings > advanced > custom styles and put this. 

```
html.hide-preferences-window div[class='x9f619 x1n2onr6 x1ja2u2z'] > div:nth-of-type(3) > div > div {
    display: block !important;
}
```

This should nullify it.

**Comment by mquevill** [comment]
This should be fixed as of [v2.60.3](https://github.com/sindresorhus/caprine/releases/tag/v2.60.3). If you're still having this issue with that version, feel free to reopen this issue.

**Comment by penzur** [comment]
I am still having this issue
